src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private final int numberofBall = new Random().nextInt(100);
```

Comment:
1. `100` is magic number
2. number of ball should be calculated every time when getRandomBall is being called